src/Xamarin.Android.Build.Tasks/Tasks/Generator.cs

```diff
@@ -147,7 +147,7 @@ protected override string GenerateCommandLineCommands ()
 		}
 
 		protected override string ToolName {
-			get { return OS.IsWindows ? "generator.exe" : "generator"; }
+			get { return "generator.exe"; }
```

Comment:
`ToolTask` uses `System.Diagnostics.Process`, which checks to see if `ProcessStartInfo.FileName` is a managed exe. If it is, the current mono is used.

Meaning *so long as* we use `Process` to create the process, we don't need a wrapper script. It Just Works.

(Which is why we still need the `mono-symbolicate` script, as that's *not* done through `Process` but through `<Exec/>`, which doesn't have the special-cased managed code logic.)